### DISCUSSION TITLE
Csv Import from webservices

### DISCUSSION THREAD

**Discussion by WpSlt** [PrestaShop/PrestaShop#32568] [Q&A] [answered]
Is It possible to call PS CSV Import from webservicies?

**Comment by matks** [comment, marked as answer]
You mean send a CSV using a cURL request and have it processed by the Webservice? I don't think 🤔

**Reply by WpSlt** [reply]
I'm thinking at something like upload csv file in adminimport folder and then trigger built- in import action by webservice.
There is a way to achive this?

**Reply by matks** [reply]
No I don't think so. The webservice is data centric: it allows you to directly modify the data model stored in SQL. It does not perform actions like in the aback-office.
